client/src/components/Draftail/CommentableEditor/CommentableEditor.js

```diff
@@ -201,14 +237,45 @@ function CommentableEditor({commentApp, fieldNode, contentPath, rawContentState,
 
     return <DraftailEditor
     ref={editorRef}
-    onChange={setEditorState}
+    onChange={(editorState) => {
+      let newEditorState = editorState;
+      if (['undo', 'redo'].includes(editorState.getLastChangeType())) {
```

Comment:
So comment additions / removals can’t be undone or redone?

Reply:
You can undo adding the highlight, reverting the new comment to a field level comment. But the actual comment needs to be deleted in the comments system itself, which I don't think is a huge burden (we could look at doing this in specific cases - eg your comment is still in "creating" mode - but given that I don't think we can get this to work for deletions, I'm a bit cautious here)